vdist/defaults.py

```diff
@@ -1,7 +1,8 @@
 import os
 import sys
 
-PYTHON_BASEDIR = '/opt/vdist-python'
+PYTHON_BASEDIR = '/opt'
+COMPILED_PYTHON_FOLDER_NAME = 'vdist-python'
```

Comment:
Fixed. Actually I've removed that default variable because when I tried to refactor I realized that var was no longer used anywhere in code. I've runned tests and everything works OK without that variable.